src/sentry/lang/native/appconnect.py

```diff
@@ -156,14 +158,26 @@ def from_json(cls, data: Dict[str, Any]) -> "AppStoreConnectConfig":
         return cls(**data)
 
     @classmethod
-    def all_for_project(cls, project: Project) -> "List[AppStoreConnectConfig]":
+    def from_option(cls, raw_sources_option: str) -> "List[AppStoreConnectConfig]":
+        """Creates a list of new instances from a raw serialized JSON list of symbol sources,
+        typically fetched directly from a project's ``sentry:symbol_sources`` option. Useful when
+        App Store Connect symbol sources need to be updated across multiple projects.
+
+        :raises InvalidConfigError: if the data does not contain a valid App Store Connect
+           symbol source configuration.
+        """
+
         sources = []
```

Comment:
This is now a bit redundant, might as well not declare this

Reply:
removed the helper entirely since it wasn't being used 👍